ubuntu/16.04/usr/local/share/container/baseimage-10.sh

```diff
@@ -23,7 +23,7 @@ do_build() {
 }
 
 check_development_start() {
-  if [ "$DEVELOPMENT_MODE" -eq 0 ]; then
+  if [ "$DEVELOPMENT_MODE" == 'true' ]; then
```

Comment:
Does that mean that the value need to be the `true` string and can't be the integer `1` for example? As I don't see any call to convert this value?